docs/book/modules/concepts/pages/index.adoc

```diff
@@ -0,0 +1,88 @@
+
+== Advisory
+
+An advisory is an opinion about a vulnerability.
+
+These opinions include the context to which the opinions apply.
+These opinions include evaluation of the severity and scoring of a vulnerability within that context, such as CVSS scores.
+
+As mentioned above, a *CVE Record* from the CVE Project is a low-value advisory that mentions the vulnerability and provides a base opinion about it.
+It may include CVSS scores, within the context of the abstract origin containing the vulnerability.
+This may be simply in reference to the vulnerability *as it exists in source-code form*.
+
+Other, more-involved stakeholders (product vendors, upstream project owners) may issue *additional* advisories.
+These opinions may be in reference to *concrete* shipped products, contextualized to how the vulnerable code is *actually used*.
+
+== SBOM
+
+An SBOM is a source-of-someone's-truth about "what's inside it?", so
+everything in our DB is ultimately sourced from some
+source-of-truth. We can't really say definitively "product X is
+composed of A1, A2 + A3". Instead, we can have multiple simultaneous
+statements -- SBOM's -- from multiple people saying "product X is
+claimed by Bob to be A1 + A2" and "product X is claimed by Jim to be
+A1 + A97". So an SBOM is the entity to track the origin of the
+supposed "evidence" of assertional statements about products... about
+packages... about vulnerabilities...
+
+== Package
+
+A package is an atomic artifact or component.
+Packages may be addressed using pURLs.
+A package may be described by an SBOM describing how it is created and its contents.
+A package may certainly contain other packages (e.g. shading one Java jar into another).
+A package may also be the sole member of a Product (`UBI-8.0.13-x86.oci` may be the singular package within the "UBI 8.0.13-x86" product).
+A package is one step more abstract than an *artifact*.
+
+=== pURL
+
+Package URLs (pURLs) are possibly ambiguous names applied to packages.
+A simple pURL such as `pkg:maven/org.apache/log4j@1.2.3` may or may not refer to a unique artifact.
+With additional qualifiers, it is possible to produce a URI that asserts uniqueness, such as `pkg:maven/org.apache/log4j@1.2.3?repository_url=repo.jboss.com`.
+Without additional qualifiers, the implicit aspects (such as `repository_url`) must be taken into account.
+For instance, an unqualified `pkg:maven` pURL *implies* "the jar from Maven Central, and none other".
+
+== Product
+
+A product is a *named collection of 1 or more packages* for a concrete shippable thing.
+
+Products may be addressed using CPEs or some other future identification method.
+A product may be described by an SBOM describing its components, which may be other products or packages, or their SBOMs.
+
+NOTE: Given Red Hat ProdSec definitions, grouping of Products may need to occur within some sense of Product Versions, or Product Streams.
```

Comment:
Using the word "ProdSec" is a slang term, use "Product Security" instead.  Also, the words "Product", "Version", and "Streams" do not need to be capitalized.

For example: `Given Red Hat Product Security definitions, the grouping of products might need to occur within some sense of the product versions, or the product streams.`